frontends/bmd/src/pages/poll_worker_screen.tsx

```diff
@@ -178,15 +196,36 @@ function PrecinctScannerTallyReportModal({
         copies,
       }
     );
+    const reportTitle = getPollsReportTitle(
+      precinctScannerTally.pollsTransition
+    ).toLowerCase();
+    const reportPrecinctName = getPrecinctSelectionName(
+      electionDefinition.election.precincts,
+      precinctScannerTally.precinctSelection
+    );
+    await logger.log(LogEventId.TallyReportPrinted, 'poll_worker', {
+      disposition: 'success',
+      message: `Printed ${copies} copies of a ${reportTitle} for ${reportPrecinctName} exported from scanner ${precinctScannerTally.machineId}.`,
+      scannerPollsTransition: precinctScannerTally.pollsTransition,
+      timePollsTransitionedOnScanner:
+        precinctScannerTally.timePollsTransitioned,
+      timeReportSavedToCard: precinctScannerTally.timeSaved,
+      totalBallotsScanned: precinctScannerTally.totalBallotsScanned,
```

Comment:
Not sure if there's other information I should be included in this log, or if I should also be including this metadata when logging the _clearing_ of the card.

Reply:
I think just having the metadata here is fine